### ISSUE TITLE
Unable to clear the Roles field in Edit User

### ISSUE THREAD

**Issue by dougjuk** [nabeelio/phpvms#939]
**Describe the bug**
If you add a role to a User and save it you are unable to go back and remove the role from the user.  The field does not like to be set to an empty state.

**Version**
7.0.0-dev

**To Reproduce**
1) Edit a User allocating a role for example Flight Editor  (Created in Roles) and save.  He has access to flights
2) Now go back into the same user and press the little x to delete that role and save.
3) Go back into the User again you will find the he still has flight editor role.
 
It appears that once you save an entry in this field you cannot save it empty.
To get around this and remove access from a user I created a Role with no permissions called Null user then added and saved that.

**Expected behavior**
You should be able to remove the role from that field and save it.

**Comment by nabeelio** [comment]
Is this still open?

**Comment by dougjuk** [comment]
As far as I am aware there has been no fix.  I will try to test later but at the moment I have a null user with no permissions.

The only other way would be to go into the database and set back to null - I know that's a bad move.

regards
Doug

**Comment by dougjuk** [comment]
Hi Nabeel,  Just to confirm this still exists.  I think there are other fields like this in the system where once you have entered you cannot remove the entry.  Tried it today with new user.  Every time I deleted the role and saved it would still be there when I looked again.  I added to do not deactivate role we have now that cleared it but you cannot remove that now.  So I used my null user again.  So it is still a problem.
